### DISCUSSION TITLE
An error ocurred while making the request

### DISCUSSION THREAD

**Discussion by hellofinch** [wavelog/wavelog#1059] [Q&A] [answered]
Hello, me again.
I import my logbook with adfi file. And I can see my log in web. But when I click logbook->advanced. An error pop up. no QSOs show up.
How can I find the web log to figure out what happened? Maybe some configuration with error?
![image](https://github.com/user-attachments/assets/f3e05ba7-45f2-4cab-94c0-99d07e5b104f)
And as you can see, my log has been imported successfully.
![image](https://github.com/user-attachments/assets/efc17d8c-acbf-4638-bee0-a44fe9f69d25)

:)

**Comment by phl0** [comment]
Please have a look for errors on the JS console of the browser and show the error message.

**Reply by hellofinch** [reply]
Thanks!!
![image](https://github.com/user-attachments/assets/665dc325-b751-458e-80dc-780cf6f2fba9)
Here is the snapshot. I think it's something in an infinity loop.
Should I reinstall the wavelog again? :)

**Comment by phl0** [comment]
No. Since when does the error occur? And what do you do to reproduce it? Just call the logbook advanced?

**Reply by hellofinch** [reply]
Thanks for your reply.
Before you mentioned the console log, I just call the logbook advanced and the error dialog show.
 When I open the console, I see the loop log without any operation.

**Reply by hellofinch** [reply]
I reinstalled the wavelog and the error shows up without any operations.

**Reply by phl0** [reply]
> I reinstalled the wavelog and the error shows up without any operations.

I cannot reproduce the error here. Did you clear the browser cache? Can you provide the ADIF file that you imported? Maybe I can use that to reproduce the error.

**Reply by phl0** [reply]
And could you also please show the network part of the browser console with the request that fails and the webserver's answer in that failed request?

**Reply by hellofinch** [reply]
OK. I will show the network part of browser console.
I put the har file here.
[wavelog.zip](https://github.com/user-attachments/files/17327995/wavelog.zip)
BTW, I used Nginx as the web server. And we discussed the error [there](https://github.com/wavelog/wavelog/discussions/917).
I follow the [issue](https://github.com/wavelog/wavelog/issues/1011) to fix the problem.
I think this is the result of the loop.
I installed the wavelog with apache2 on another device too and everything worked fine.
Maybe you can try to reproduce with the nginx. more details about nginx with wavelog you can find [here](https://github.com/wavelog/wavelog/discussions/917) and [here](https://github.com/wavelog/wavelog/issues/1011)
:)

**Reply by AndreasK79** [reply]
@hellofinch the .har file doesn't seem to contain when you enter the advanced logbook. We are after the call to search under the network tab, and I could not locate that in your file. Could you please check that?

**Reply by AndreasK79** [reply]
Here is an example: 
![image](https://github.com/user-attachments/assets/5387f6d3-bdf1-4b0c-b33a-e5cb0728aaac)
If you click on that call, and then go to response, it should contain an error.

**Reply by hellofinch** [reply]
Thanks for your reply. 

> Thanks for your reply. Before you mentioned the console log, I just call the logbook advanced and the error dialog show. When I open the console, I see the loop log without any operation.

I mentioned here.
There may have been some issues with my previous explanation. Before mentioning the browser console, I hadn’t noticed whether there was an infinite loop issue. When I opened the browser console, I found that the infinite loop occurred even without any actions on my part. So I do nothing before I export the har file. I will provide my adif file here as well. Therefore, my call is not visible in the har file. I will provide my adfi file here as well.
[bi1uvz.403560.20240812073006.zip](https://github.com/user-attachments/files/17336744/bi1uvz.403560.20240812073006.zip)

**Comment by hellofinch** [comment, marked as answer]
HAHA!
It solved.
The nginx configuration makes the issue.
More info can find [here](https://github.com/wavelog/wavelog/issues/1011).
73 :)

**Comment by iw0hnb** [comment]
Hi all, 
I've recently upgraded my standalone installation of weavelog to the latest version 2.0.5. (running on apache webserver  on dietpi - no nginx  and no docker ).
Everything looks fine but when I try to open LOGBOOK->ADVANCED it shows an error (see images).
Can someone help me ? 
Thank you
Stefano
IW0HNB
![2025-07-08_12-51](https://github.com/user-attachments/assets/ed512299-f79c-466a-9358-bc235ea54f7c)
![2025-07-08_12-51_1](https://github.com/user-attachments/assets/a70a1d38-a583-4f0a-b3ce-4412ab050d1b)
![2025-07-08_13-00](https://github.com/user-attachments/assets/64831f6b-3129-44f9-ba04-51415f70b2f0)

**Reply by AndreasK79** [reply]
Hi, this is not the same error. The problem is fixed in the dev branch. Either swap to that, or wait for a new release (should be in the next few days).

**Reply by AndreasK79** [reply]
I should say not the same reason for this error.

**Reply by iw0hnb** [reply]
thank you very much! I forgot to say the issue shows up only if upgrading from previus release, a fresh new installation isn't affected by this problem, however i couldn't find a simple way to restore all my data and settings.
I'll be waiting for the new release.
Thank you
Stefano

**Reply by AndreasK79** [reply]
I see. The problem is related to missing DXCCdata. You can always go to "Update country files" and run "Check QSOs missing DXCC data" to see if that helps.

**Comment by iw0hnb** [comment]
Hi, thanks for the support, however it does not work...
![2025-07-08_17-50](https://github.com/user-attachments/assets/1d091248-4405-4589-a3e2-58c441137dd7)

**Reply by iw0hnb** [reply]
HI all, as suggested I've updated to the latest release and everything is back to normal! No more issues! Thank you!
